benchmark.py

```diff
@@ -73,6 +83,32 @@ def TryFunctionWithTimeout(func, error_handler, num_tries,
   raise BenchmarkError("All tries failed.")
 
 
+def AppendArgument(main_command, boolean_var, arg_name,
```

Comment:
I would change this to `GetBooleanFormattedArgument` and drop the append part (just return a string with the arg, let the caller `+=` it). That would be more functional and avoid baking in policy in the function.